Neos.Neos/Resources/Private/Translations/en/Modules.xlf

```diff
@@ -2,15 +2,6 @@
 <xliff version="1.2" xmlns="urn:oasis:names:tc:xliff:document:1.2">
 	<file original="" product-name="Neos.Neos" source-language="en" datatype="plaintext">
 		<body>
-			<trans-unit id="clickToEdit" xml:space="preserve">
-				<source>Click to edit</source>
-			</trans-unit>
-			<trans-unit id="clickToDelete" xml:space="preserve">
-				<source>Click to delete</source>
-			</trans-unit>
-			<trans-unit id="cancel" xml:space="preserve">
-				<source>Cancel</source>
-			</trans-unit>
```

Comment:
Yeah, you are right. I slightly misunderstood the structure for the translations and thought, the folder is relevant for the templates, and the files are only for the overview and structure of the developer. I didn't know that they can be referenced.
I found some occurrences and will put the labels back again. Thanks for the hint!